### ISSUE TITLE
[Incorrect behavior][3.1.1]: False Conflicts Error

### ISSUE THREAD

**Issue by EggoTheSquirrel** [Raytwo/ARCropolis#246]
### Have you read the disclaimer before proceeding?

- [X] I have read the disclaimer.

### System Firmware Version

14.0.0

### Atmosphere Version

1.3.0

### Smash Version

13.0.1

### ARCropolis Version

3.1.1

### Version channel

Stable

### SD card formatting

FAT32

### Can this incorrect behavior be consistently reproduced?

Yes

### Expected behavior

I was expecting Smash to boot up with no file conflicts, as there are currently none in my `sd:/ultimate/mods` directory. 
Instead, I keep getting an error stating that two mods that are not enabled and are _not even on my sd card anymore_ are conflicting. This means that I am unable to boot into the game.

The conflicts.json reads exactly as follows:
```
{
  "fighter/trail/model/body/c06/def_trail_003_col.nutexb": [
    "sd:/ultimate/mods/.(SKIN) Purple Sora (c06)",
    "sd:/ultimate/mods/(SKIN) Black Star Sora (c06)"
  ],
  "stage
```

Neither of the directories listed exist. 

### Context

I had simply been updating my mod software, atmosphere and arcropolis to the latest version.

### Configuration

arcropolis.toml:
```
[infos]
version = "2.1.1"

[paths]
arc = "rom:/arc"
umm = "sd:/ultimate/mods"
extra_paths = []

[updater]
server_ip = "upd.arcropolis.com"
beta_updates = false

[logger]
logger_level = "Off"

[misc]
debug = false
region = "us_en"
```

**Comment by Raytwo** [comment]
That doesn't sound right, you use ARCropolis 3.1.1 but your configuration file is still from 2.1.1, so which is it?

**Comment by EggoTheSquirrel** [comment]
I thought I had updated to 3.1.1, I guess I did it wrong? I copied over all the files from the release to my arcropolis folder, I don't know why my config file is outdated. (or how I missed that when copying it over). I can check when I get back to my setup tomorrow night; I'm not home right now.

**Comment by EggoTheSquirrel** [comment]
OK, so I just got back to my home system, and it threw an entirely different conflicts error that referenced two actual mods I had installed. I removed one of those mods, and now it boots fine. I guess the problem wasn't that I was getting an error where there was none, but that I was getting the wrong error message. If I had just noticed the real error it could have all been avoided. thanks for your help!
